.github/workflows/publish-dist.yml

```diff
@@ -31,10 +31,9 @@ jobs:
           set -ex
           mkdir dist-repo
           cd dist-repo
-          tar -xf ../artifact/webpack-jumpstart.tar dist package-lock.json .sha256sums || true
+          tar -xf ../artifact/webpack-jumpstart.tar dist .sha256sums
```

Comment:
We may not use package-lock.json in workflows any more, but it's still an useful file to look at/diff as a developer, and ensures that "regular" web developers not familiar with our cache magic can work with the tree. So I'd rather leave this out.

Reply:
We discussed that this was a misunderstanding: `package-lock.json` remains in the `node_modules` directory, but this is the `dist/` cache we discuss here.

Reply:
Indeed, sorry for the thinko!